source/Ship.cpp

```diff
@@ -492,6 +492,17 @@ void Ship::Load(const DataNode &node)
 			description += child.Token(1);
 			description += '\n';
 		}
+		else if( key == "remove" && child.Size() >= 2) {
+			if(variantName.empty())
+			{
+				child.PrintTrace("Skipping invalid 'remove' on a non variant:");
+				continue;
+			}
```

Comment:
Let's make this work for non-variant as well. This would be useful for plugins that want to remove bays from existing ships.

Reply:
limitation removed